### PR TITLE
Add centos-stream package to specfile packit.yaml

### PR DESCRIPTION
**Pull request by majamassarini**
To be able to propose-downstream using packit
follow this guide: https://gitlab.com/CentOS/Integration/packit-cbs/docs

[This](https://gitlab.com/CentOS/Integration/packit-cbs/rpms/python-specfile/-/merge_requests/2) has been opened with packit

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -33,6 +33,32 @@ packages:
         - bash -c "ls -1t ./fedora/*.tar.gz | head -n 1"
       get-current-version: python3 -m setuptools_scm
 
+  specfile-centos-stream:
+    upstream_package_name: specfile
+    downstream_package_name: python-specfile
+    specfile_path: &specfile_path_centos centos-stream/python-specfile.spec
```
Comment (nforro) [inline .packit.yaml:39]: This path doesn't seem to exist.

File: .packit.yaml [hunk 2]
```diff
@@ -33,6 +33,32 @@ packages:
         - bash -c "ls -1t ./fedora/*.tar.gz | head -n 1"
       get-current-version: python3 -m setuptools_scm
 
+  specfile-centos-stream:
```
Comment (nforro) [inline .packit.yaml:36]: I'm a bit uncertain about the `centos-stream` name, it's rather ambiguous. When I hear CentOS Stream I think https://gitlab.com/redhat/centos-stream/rpms, and we also refer to it as such in our [docs](https://packit.dev/docs/configuration/upstream/propose_downstream#syncing-the-release-to-centos-stream). What about something like `specfile-centos-integration-sig`?

File: .packit.yaml [hunk 3]
```diff
@@ -33,6 +33,32 @@ packages:
...
+      - python3-build
+      - python3-setuptools_scm
+    actions:
+      pre-sync:
+        - python3 plans/git_reference.py
+      create-archive:
+        - python3 -m build --sdist --outdir ./fedora/
+        - bash -c "ls -1t ./centos-stream/*.tar.gz | head -n 1"
+      get-current-version: python3 -m setuptools_scm
+    dist_git_base_url: https://gitlab.com # this is not taken into account by centpkg-sig yet!
```
Comment (nforro) [inline .packit.yaml:57]: > this is not taken into account by centpkg-sig yet!

So this needs some changes in Packit as well for these options to be passed to `centpkg-sig` (or the configured `pkg_tool` in general)?

Comment (majamassarini) [reply .packit.yaml:57]: Yes exactly, this is not working yet. Without changes in packit and centpkg-sig.

**Review by nforro — APPROVED** [review]

**Pull request merged**